tests/network-tests/src/flows/proposals/electionParametersProposal.ts

```diff
@@ -7,17 +7,18 @@ import { Resource } from '../../Resources'
 
 // Election parameters proposal scenario
 export default async function electionParametersProposal({ api, lock }: FlowProps): Promise<void> {
-  const debug = Debugger('flow:electionParametersProposal')
+  const debug = Debugger('integration-tests:flow:electionParametersProposal')
   debug('Started')
   await lock(Resource.Proposals)
-
+  debug('lock acquired contining flow')
```

Comment:
Not sure if we still need those?

Reply:
I missed a couple of those while I was doing "console.log" debugging :)